core/src/main/resources/hudson/model/Api/index.jelly

```diff
@@ -98,58 +98,47 @@ THE SOFTWARE.
         -->
       </dl>
 
-      <p>
-        For more information about remote API in Jenkins, see
-        <a href="https://jenkins.io/redirect/remote-api">the documentation</a>.
-      </p>
+      <p>For more information about remote API in Jenkins, see <a href="https://jenkins.io/redirect/remote-api">the documentation</a>.</p>
```

Comment:
@recena It's not a big deal if it happens occasionally. We also don't expect you to not do an obvious improvement near the code you need to change anyway (let's say "near" is "`git diff` doesn't skip unmodified lines in between" as a rule of thumb). But IIRC inclusion of unrelated changes, even in otherwise unchanged files, has been a recurring issue in PRs from you. We bring it up every time it happens, but you don't internalize our feedback. In fact, it looks deliberate, despite being asked not to do it.

What would you expect us to do in this situation?